roles/openshift-applier/tasks/process-helm.yml

```diff
@@ -28,24 +29,19 @@
     helm_values_option: "{{ helm_values_option }} --values '{{ item }}'"
   loop: "{{ helm.values_param | default([]) }}"
 
-- name: "{{ oc_action | capitalize }} OpenShift objects based on helm template for '{{ entry.object }} : {{ content.name | default(helm.name) }}'"
+- name: "Helm install chart to target Openshift cluster: '{{ entry.object }} : {{ content.name | default(helm.name) }}'"
   shell: >
-    helm template \
+    helm install \
        {{ helm.name }} {{ helm.chart }} \
        {{ helm_version_option }} \
        {{ helm_namespace_option }} \
+       {{ helm_timeout_option }} \
        {{ helm_set_option }} \
        {{ helm_values_option }} \
        {{ helm.flags|d('') }} \
-       | \
```

Comment:
but, yes, this is probably why you split it into two before - let me think about this a bit more

Reply:
I was thinking about constructing entire helm command, including all flavours,  before the execution step - but decided to leave that aside as a future improvement task

Reply:
PR submitted: https://github.com/jfilipcz/openshift-applier/pull/1